src/main/java/org/broadinstitute/consent/http/resources/VoteResource.java

```diff
@@ -87,10 +83,56 @@ public Response updateVotes(
                 return createExceptionResponse(new NotFoundException());
             }
 
-            List<Vote> updatedVotes = voteService.updateVotesWithValue(votes, vote, rationale);
+            List<Vote> updatedVotes = voteService.updateVotesWithValue(votes, voteUpdateInfo.getVote(), voteUpdateInfo.getRationale());
             return Response.ok().entity(updatedVotes).build();
         } catch (Exception e) {
             return createExceptionResponse(e);
         }
     }
+
+    static class VoteUpdateInfo {
+
+        @JsonProperty
```

Comment:
As a side note, `@JsonProperty` is mostly unnecessary. This is a Jackson annotation that allows for writing/parsing via Jackson libraries. Since we're using Gson here, we don't need them. We should have a tech debt story to deprecate this usage.